pages/keplr/home-page.js

```diff
@@ -0,0 +1,13 @@
+const tokenNameLabel = tokenName => `:text-is("${tokenName}")`;
+const tokenParentSelector = '../../../..';
+const selectAllTokensCheck = 'input[type="checkbox"]:enabled';
+const newTokensFound = 'new token(s) found';
+const addChainsButton = 'Add Chains';
```

Comment:
I was seeing Keplr but didn't find any text saying `new token(s) found on Home URL` 
Can you guide me on testing this workflow?

Reply:
test is added on this. you can get an idea from there how the flow occurs